app/src/main/java/com/epfl/beatlink/ui/map/GoogleMapView.kt

```diff
@@ -26,26 +35,34 @@ import com.google.maps.android.compose.MarkerState
 import com.google.maps.android.compose.rememberCameraPositionState
 import kotlinx.coroutines.launch
 
+val mutuableList = MapUsersList.mutableList
```

Comment:
Consider renaming this variable to improve code readability and better reflect its purpose.